frontend/src/components/Documentation.vue

```diff
@@ -156,12 +156,21 @@
             <span class="icon pr-5 is-large has-text-info"><i class="fa fa-connectdevelop"></i></span>
```

Comment:
Maybe it looks better to increase the gap between the text and the demo video.

Reply:
Sound good.